eras/conway/impl/test/Test/Cardano/Ledger/Conway/BinarySpec.hs

```diff
@@ -28,4 +30,9 @@ spec = do
     roundTripCborSpec @(Voter StandardCrypto)
     roundTripConwayCommonSpec @Conway
     -- ConwayGenesis only makes sense in Conway era
-    roundTripEraSpec @Conway @(ConwayGenesis StandardCrypto)
+    context "Genesis" $ do
+      roundTripCborSpec @(UpgradeConwayPParams Identity)
+      roundTripCborSpec @(Constitution (ConwayEra StandardCrypto))
```

Comment:
This type is tested in Shelley, where it is defined

Reply:
Noted! :slightly_smiling_face: